README.md

```diff
@@ -1,5 +1,37 @@
 # Handling images in PDF file using C#
 PDF has become one of the most popular and widely used file formats, here images play an important role in PDF documents, images can grab our attention easily. So, in this article, you will learn how to add, extract, remove, and replace images from PDF documents using C#.
 
-## Blog reference
-[Add, Remove, Extract, and Replace Images in PDF using C#](https://www.syncfusion.com/blogs/post/add-remove-extract-and-replace-images-in-pdf-using-csharp.aspx)
+Sample name | Description
+--- | ---
+[Add image](https://github.com/SyncfusionExamples/pdf_image_handling/blob/master/Add_image) | To add an image to a PDF file in console application.
+[Extract image](https://github.com/SyncfusionExamples/pdf_image_handling/blob/master/Extract_images) | To extract all the images from a PDF file in console application.
+[Remove image](https://github.com/SyncfusionExamples/pdf_image_handling/blob/master/Remove_images) | To remove images from a PDF document in console application. 
```

Comment:
Remove images from an existing PDF document

Reply:
changed the description